packages/utils/lib/types/common.ts

```diff
@@ -75,6 +75,7 @@ export type FormItemType =
   | 'Checkbox'
   | 'Header 3'
   | 'Header 4'
+  | 'Html'
```

Comment:
can you capitalize HTML please

Reply:
acknowledged with amend*, also enabled Markdown for the `Description` as we previously had done together